### DISCUSSION TITLE
Implementation of Towboot

### DISCUSSION THREAD

**Discussion by ghost** [raspiblitz/raspiblitz#3918] [Feature / App Ideas]
Is there some plan to implement towboot for the image of raspiblitz?

**Comment by openoms** [comment]
Please provide some more info if this is a feature request.

**Comment by ghost** [comment]
https://tow-boot.org/

https://github.com/Tow-Boot/Tow-Boot/

It is a uniformization of the firmware on arm architectures, in order to help distro developers to not coustomize the uboot for each different device. For what I understand, it would benefit raspiblitz for being much easier to install in other single board computers, while making your amazing work easier.

**Comment by rootzoll** [comment]
Added to v1.8.3 milestone to have a deeper look into
